transform_test.go

```diff
@@ -170,9 +171,15 @@ func TestInjectNamespaceAPIService(t *testing.T) {
 	assert(f.Resources()[0], "food")
 }
 
-func TestInjectNamespaceCRD(t *testing.T) {
+func TestInjectNamespaceCRDV1Beta1(t *testing.T) {
```

Comment:
If we're supporting both versions, why rename the function?

Reply:
Thanks for your reply, I have corrected the code